packages/vue_frontend_rpc/src/components/Selector.vue

```diff
@@ -1,137 +0,0 @@
-<template>
-  <v-main id="selector-component" v-if="selector">
```

Comment:
this component wasn't in use?

Reply:
we are not using this component anymore. it was change made by Alex